README.md

```diff
@@ -56,6 +62,27 @@ Or, if you prefer the old style:
 python3 setup.py install
 ```
 
+
+## Minimal Example
+To test the installation run this for example in ipython
+
+```
+from rasputin import triangulate_dem
+
+vs = triangulate_dem.PointVector([(1,0,0), (0,1,0), (0,0,0), (0.25,0.25,1)])
+points, faces = triangulate_dem.lindstrom_turk_by_ratio(vs, 2.0)
+for f in faces:
+   print(f)
+```
+
+This should print out
+```
+>> (3, 1, 2)
+>> (0, 1, 3)
+>> (0, 3, 2)
+```
+Congratulations you triangulated a small mountain
```

Comment:
Please improve the grammar :)

Reply:
Updated the grammer, hope it suits the king!